index.html

```diff
@@ -0,0 +1,182 @@
+                        <h6 class="t-title">Forum</h6>
+                        <p class="text-white p-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus ea sit
+                            error distinctio vitae.</p>
+                    </div>
+
+                    <div
+                        class="col-lg p-3 bg-cards d-flex flex-lg-column justify-content-center align-items-center gap-2">
+                        <img class="icons" src="src/img/destination.png" alt="">
+                        <h6 class="t-title">Workshop</h6>
+                        <p class="text-white p-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus ea sit
+                            error distinctio vitae.</p>
+                    </div>
+
+                    <div
+                        class="col-lg p-3 bg-cards d-flex flex-lg-column justify-content-center align-items-center gap-2">
+                        <img class="icons" src="src/img/app.png" alt="">
+                        <h6 class="t-title">CC Ignite</h6>
+                        <p class="text-white p-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus ea sit
+                            error distinctio vitae.</p>
+                    </div>
+                </div>
+            </div>
+            <div class="d-flex justify-content-center mt-5 pb-5">
+                <a class="text-white text-center" href="#">SEE THE WHOLE PROGRAM</a>
+            </div>
+        </section>
+
+        <section class="container mt-5 d-flex flex-column">
+            <h2 class="text-center">Feature speakers</h2>
+            <div class="vector-container d-flex justify-content-center mb-4">
+                <div class="vector"></div>
+            </div>
+            <div class="cards-container row d-flex justify-content-center mt-5 mx-0">
+                
+            </div>
+            <button id="see_more" class="btn" type="button">SEE MORE<i class="fas fa-chevron-down"></i></button> 
+        </section>
+
+        <section class="companies py-5">
+            <h2 class="text-center text-secondary">Partner</h2>
+            <div class="vector-container d-flex justify-content-center mb-4">
+                <div class="vector"></div>
+            </div>
+            <ul class="d-lg-flex justify-content-center align-items-center gap-5">
+                <li class="li-companies text-center mt-4"><img src="src/img/mozilla.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/google.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/naver.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/daumkakao.png" alt=""></li>
+                <li class="li-companies text-center mt-4"><img src="src/img/airbnb.png" alt=""></li>
+            </ul>
+        </section>
```

Comment:
The partner section in the mobile version should look almost identical to the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

Your design:
![image](https://user-images.githubusercontent.com/60292761/147277537-4d145326-18ac-4f24-aff4-861cbd416328.png)

The original design : 
![image](https://user-images.githubusercontent.com/60292761/147277574-c6d3783b-cad0-41b5-8aec-8e2d47c688e5.png)

Please make sure to follow the original design as you can just make it horizontally.